util/tej-attributions.ts

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
We don't need this file or this attribution because it's creative commons, meaning we can use it without attribution AFAIK.

Reply:
Aye, Aye. How are you doing today?